security/pkg/server/ca/server_test.go

```diff
@@ -531,7 +531,7 @@ func TestGetServerCertificate(t *testing.T) {
 	}
 
 	server, err := New(ca, time.Hour, false, []string{"localhost"}, 0,
-		"testdomain.com", true, jwt.JWTPolicyThirdPartyJWT)
+		"testdomain.com", true, jwt.JWTPolicyThirdPartyJWT, "kubernetes")
```

Comment:
Like a previous comment, using "kubernetes" as an ID is confusing.

Reply:
I am using `kubernetes` as default value in testing to align with what we have in runtime:
```   - name: ISTIO_META_CLUSTER_ID
              value: "{{ .Values.global.multiCluster.clusterName | default `Kubernetes` }}"```